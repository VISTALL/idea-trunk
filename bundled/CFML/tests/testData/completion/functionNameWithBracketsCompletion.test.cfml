<cffunction name="mineFunction"></cffunction>
<cfscript>
mine<caret>
</cfscript>
