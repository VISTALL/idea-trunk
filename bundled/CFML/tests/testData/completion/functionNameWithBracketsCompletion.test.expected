<cffunction name="mineFunction"></cffunction>
<cfscript>
mineFunction
</cfscript>
